tools/wptrunner/wptrunner/browsers/edge.py

```diff
@@ -6,6 +6,9 @@
                                           SeleniumRefTestExecutor)  # noqa: F401
 from ..executors.executoredge import EdgeDriverWdspecExecutor  # noqa: F401
 
+import subprocess
+from tools.wpt.utils import call
```

Comment:
Let's not rely on `tools.wpt` here; just use `subprocess.check_output` without that wrapper. (I think there's still some use of wptrunner with it actually installed, and being able to import this relies on `tools` being on the path.)

Reply:
Using subprocess.check_output.